src/main/java/com/autotune/analyzer/serviceObjects/Converters.java

```diff
@@ -61,42 +61,16 @@ public static KruizeObject convertCreateExperimentAPIObjToKruizeObject(CreateExp
                 List<KubernetesAPIObject> kubernetesAPIObjectsList = createExperimentAPIObject.getKubernetesObjects();
                 for (KubernetesAPIObject kubernetesAPIObject : kubernetesAPIObjectsList) {
                     K8sObject k8sObject = null;
-                    // Verify the experiment type.
-                    // If the experiment type is null, default is container type experiment.
-                    // TODO: Update to make this field mandatory and validate if it is a container type.
-                    if (null == kubernetesAPIObject.getExperimentType() || kubernetesAPIObject.getExperimentType().equalsIgnoreCase(AnalyzerConstants.ExperimentTypes.CONTAINER_EXPERIMENT)) {
+                    String experimentType = createExperimentAPIObject.getExperimentType();
+                    // check if exp type is null to support remote monitoring experiments
+                    if (null == experimentType || AnalyzerConstants.ExperimentTypes.CONTAINER_EXPERIMENT.equalsIgnoreCase(experimentType)) {
```

Comment:
use if (ko.isContainerExperiment()) {

else if (ko.isNamespaceExperiment()) {

Reply:
look for ContainerAPIObJect comment

Reply:
Implemented the interface as discussed.